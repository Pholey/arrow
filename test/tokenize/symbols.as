// 1-character
+
-
/
*
%
&
|
^
!
=
<
>
.
:
;
{
}
(
)
[
]
,

// 2-character
+=
-=
*=
/=
%=
&=
|=
^=
==
!=
>=
<=

// Mixed
....
++=
--=
==||=
