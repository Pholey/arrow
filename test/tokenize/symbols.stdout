test/tokenize/symbols.as:2:1: symbol: `+`
test/tokenize/symbols.as:3:1: symbol: `-`
test/tokenize/symbols.as:4:1: symbol: `/`
test/tokenize/symbols.as:5:1: symbol: `*`
test/tokenize/symbols.as:6:1: symbol: `%`
test/tokenize/symbols.as:7:1: symbol: `&`
test/tokenize/symbols.as:8:1: symbol: `|`
test/tokenize/symbols.as:9:1: symbol: `^`
test/tokenize/symbols.as:10:1: symbol: `!`
test/tokenize/symbols.as:11:1: symbol: `=`
test/tokenize/symbols.as:12:1: symbol: `<`
test/tokenize/symbols.as:13:1: symbol: `>`
test/tokenize/symbols.as:14:1: symbol: `.`
test/tokenize/symbols.as:15:1: symbol: `:`
test/tokenize/symbols.as:16:1: symbol: `;`
test/tokenize/symbols.as:17:1: symbol: `{`
test/tokenize/symbols.as:18:1: symbol: `}`
test/tokenize/symbols.as:19:1: symbol: `(`
test/tokenize/symbols.as:20:1: symbol: `)`
test/tokenize/symbols.as:21:1: symbol: `[`
test/tokenize/symbols.as:22:1: symbol: `]`
test/tokenize/symbols.as:23:1: symbol: `,`
test/tokenize/symbols.as:26:1-3: symbol: `+=`
test/tokenize/symbols.as:27:1-3: symbol: `-=`
test/tokenize/symbols.as:28:1-3: symbol: `*=`
test/tokenize/symbols.as:29:1-3: symbol: `/=`
test/tokenize/symbols.as:30:1-3: symbol: `%=`
test/tokenize/symbols.as:31:1-3: symbol: `&=`
test/tokenize/symbols.as:32:1-3: symbol: `|=`
test/tokenize/symbols.as:33:1-3: symbol: `^=`
test/tokenize/symbols.as:34:1-3: symbol: `==`
test/tokenize/symbols.as:35:1-3: symbol: `!=`
test/tokenize/symbols.as:36:1-3: symbol: `>=`
test/tokenize/symbols.as:37:1-3: symbol: `<=`
test/tokenize/symbols.as:40:1: symbol: `.`
test/tokenize/symbols.as:40:2: symbol: `.`
test/tokenize/symbols.as:40:3: symbol: `.`
test/tokenize/symbols.as:40:4: symbol: `.`
test/tokenize/symbols.as:41:1: symbol: `+`
test/tokenize/symbols.as:41:2-4: symbol: `+=`
test/tokenize/symbols.as:42:1: symbol: `-`
test/tokenize/symbols.as:42:2-4: symbol: `-=`
test/tokenize/symbols.as:43:1-3: symbol: `==`
test/tokenize/symbols.as:43:3: symbol: `|`
test/tokenize/symbols.as:43:4-6: symbol: `|=`
test/tokenize/symbols.as:44:1: end
